### PR TITLE
Upgrade to postcss 8 and clean up dependencies

### PR DESCRIPTION
**Pull request by mgronblad**
Migrated to postcss 8 with the help of this guide: https://evilmartians.com/chronicles/postcss-8-plugin-migration

Also changed ci from travis to github actions.

Have not yet changed package version or changed to github actions-icons in the readme for ci status, I'll do that next week.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,131 @@
...
+*.lcov
+
+# nyc test coverage
+.nyc_output
+
+# Grunt intermediate storage (https://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# Bower dependency directory (https://bower.io/)
+bower_components
```
Comment (jonaswalden) [inline .gitignore:34]: why all of these entries? we neither do nor will use grunt or bower.

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+14
```
Comment (jonaswalden) [inline .nvmrc:1]: Why not 16? it's the current LTS version

**Review by jonaswalden — APPROVED** [review]
🚀

File: .gitignore (hunk 1 above)
Comment (mgronblad) [reply .gitignore:34]: It's Githubs template for node apps (https://github.com/github/gitignore/blob/main/Node.gitignore)

**Pull request merged**